### PR TITLE
Making sure test clock is wired into api tests, and updating API init invis query param

### LINKED ISSUE #122: Adding message with initialInvisibilitySeconds has no effect.
"initialInvisibilitySeconds" parameter in addMessage(...) has no effect.
See the test below:

```
        final QueueName queueName = QueueName.valueOf(QUEUE_NAME);
        final AccountName accountName = AccountName.valueOf(ACCOUNT_NAME);
        final AccountKey accountKey = AccountKey.valueOf(KEY);
        final CassieqApi client = CassieqApi.createClient("http://192.168.99.100:8080/", CassieqCredentials.key(accountName,
                accountKey));

        client.createQueue(accountName, new QueueCreateOptions(queueName)).execute();

        client.addMessage(accountName, queueName, "0").execute();
        client.addMessage(accountName, queueName, "4", 400000l).execute();
        client.addMessage(accountName, queueName, "3", 300000l).execute();
        client.addMessage(accountName, queueName, "2", 200000l).execute();
        client.addMessage(accountName, queueName, "1").execute();

        int counter = 0;

        for (int i = 0; i < 5; ++i)
        {
            final GetMessageResponse body = client.getMessage(accountName, queueName, 10000l).execute().body();
            counter += body == null ? 0 : 1;

        }

        client.deleteQueue(accountName, queueName).execute();
        // counter should be 2 but is always 5
        // it means we consumed messages with visibility in >= 200000 seconds (55 hours)
        System.out.println(" COUNTER : " + counter);

        assert counter <= 2;
```

### PR DESCRIPTION
**Pull request by devshorts**
Fixes #122

### CODE REVIEW

File: core/src/main/java/io/paradoxical/cassieq/discoverable/resources/api/v1/QueueResource.java [hunk 1]
```diff
@@ -277,7 +277,7 @@ public Response updateMessage(
                             @ApiResponse(code = 500, message = "Server Error") })
     public Response putMessage(
             @StringTypeValid @PathParam("queueName") QueueName queueName,
-            @QueryParam("initialInvisibilityTime") @DefaultValue("0") Long initialInvisibilityTime,
+            @QueryParam("initialInvisibilitySeconds") @DefaultValue("0") Long initialInvisibilityTime,
```
Comment (devshorts) [inline core/src/main/java/io/paradoxical/cassieq/discoverable/resources/api/v1/QueueResource.java:284]: This is the berg

**Comment by devshorts** [comment]
Deprecating query param and adding:

image

image

File: client/src/main/java/io/paradoxical/cassieq/api/client/CassieqApi.java [hunk 2]
```diff
@@ -23,6 +23,7 @@
 import retrofit.http.Query;
 
 import java.net.URI;
+import java.util.concurrent.TimeUnit;
```
Comment (jakeswenson) [inline client/src/main/java/io/paradoxical/cassieq/api/client/CassieqApi.java:26]: unneeded?

**Comment by jakeswenson** [comment]
✔️ LGTM

**Comment by devshorts** [comment]
Just noticed my error message isn't right cause it mentions the deprecated field twice. I'll fix that before merge

**Pull request merged**